test/src/main/java/org/apache/accumulo/test/ClientSideIteratorIT.java

```diff
@@ -104,6 +104,7 @@ public void testIntersect() throws Exception {
     IntersectingIterator.setColumnFamilies(si, new Text[] {new Text("bar"), new Text("foo")});
     csis.addScanIterator(si);
 
+    csis.close();
```

Comment:
Another case of it being closed before it is being used. I think switching to more try-with-resources syntax in these cases would fix a lot of these kinds of errors.